tests/Ethereum/TWAnySignerTests.cpp

```diff
@@ -57,6 +57,7 @@ TEST(TWAnySignerEthereum, Sign) {
 
     input.set_chain_id(chainId.data(), chainId.size());
     input.set_nonce(nonce.data(), nonce.size());
+    input.set_tx_mode(Proto::TransactionMode::Legacy);
```

Comment:
can we make legacy as default mode?

Reply:
it is the default, but I can take them out from tests

<img width="697" alt="image" src="https://user-images.githubusercontent.com/13562139/129576198-2b11edc1-6724-4ca3-9877-3293eac29d59.png">